.codecov.yml

```diff
@@ -1,4 +1,19 @@
 coverage:
   status:
-    project: no
+    project:
+      default:
+        target: 75%
+        flags: unit,integration
+      unit:
+        target: 75%
+        flags: unit
     patch: no
+  flags:
+    unit:
+      ignore:
+        - "raiden/app.py"
+        - "raiden/raiden_service.py"
+        - "raiden/ui/*"
+        - "raiden/network/proxies/*"
+        - "raiden/network/transport/*"
+        - "raiden/network/transport/**/*"
```

Comment:
Do we want to ignore this? If we leave all files in, the comparison between unit coverage and full coverage would be more meaningful.

Reply:
But why do we want to compare unit coverage to full coverage? The idea here was to ensure that unit tests cover (almost) everything where they make sense. So we exclude the parts of the code where unit testing does not make sense in general and set a high coverage goal for the rest.

Another problem is that codecov apparently doesn't support this. Even if they get their flag feature fixed, it will not support flag-specific ignore lists.